src/map/itemdb.c

```diff
@@ -347,6 +347,26 @@ static bool itemdb_in_group(struct item_group *group, int nameid)
 	return false;
 }
 
+/**
+ * Search for a group of item
+ * @param group return information about group
+ * @param nameid item group id to search
+ * @return bool true if found, false otherwise
+ */
+static bool itemdb_search_group(struct item_group *group, int nameid)
+{
```

Comment:
`itemdb->search` returns the item directly instead of a bool, I think it's better to follow the same logic and return the group that was found (`NULL` if not found).